src/styles/blocks/page.scss

```diff
@@ -0,0 +1,6 @@
+.page {
+  height: 100vh;
+  margin: 0;
+
+  @include center;
```

Comment:
You are using a mixin named `center` here. Make sure that this mixin is defined in your `mixins.scss` file and that it does what you expect. If it is not defined, or if it is defined incorrectly, it could cause issues with your layout.